packaging/ubuntu/rules

```diff
@@ -45,9 +45,9 @@ override_dh_auto_install:
 	mkdir -m 0755 -p ./debian/tracee-rules/usr/lib/tracee/
 	install -m 0755 ./dist/tracee-rules ./debian/tracee-rules/usr/lib/tracee/
 	ln -s /usr/lib/tracee/tracee-rules ./debian/tracee-rules/usr/bin/tracee-rules
-	# rules
-	mkdir -m 0755 -p ./debian/tracee-rules/usr/lib/tracee/rules/
-	install -m 0644 ./dist/rules/* ./debian/tracee-rules/usr/lib/tracee/rules/
+	# signatures
```

Comment:
You need to change packaging/ubuntu/control file as well. The package named `tracee-rules` should be named `tracee-signatures` now I believe. You can test generating packages for at least 1 distro (make -f builder/Makefile.packaging ubuntu-bin-focal) to make sure it works.

Reply:
Ok I see now the point you want to keep tracee-rules (since it will get deprecated). Lets check if packages are good..

Reply:
ok nm, works good.